scripts/run_Cu78.jl

```diff
@@ -143,6 +164,21 @@ start_config = Config(pos_cu78, bc_cu55)
 
 #histogram information
 n_bin = 100
+
+#----------------------------------------------------------------------------#
+evtohartree = 0.0367493
+
+n = 8.482
+m = 4.692
+ϵ = evtohartree*0.0370
+a = 0.25*nmtobohr
+C = 27.561
+
+pot = EmbeddedAtomPotential(n,m,ϵ,C,a)
+
+suttonchenpot = EmbeddedAtomPotential(9.0,6.0,0.0126*evtohartree,39.432,0.3612*nmtobohr)
+#----------------------------------------------------------------------------#
+
```

Comment:
It would be good to have the formula here/ meaning of parameters, perhaps reference to paper(s) where you have taken parameters from.

Reply:
Fixing this now